npm/builder.js

```diff
@@ -141,10 +141,7 @@ Builder.prototype.release = function (releaseVersion) {
     function (callback) { builder.prepareRelease(releaseVersion, callback); },
     function (callback) { builder.build(callback); },
     function (callback) { builder.runTest(callback); },
-    function (callback) { builder.copyToDist(callback); },
     function (callback) { builder.commit(releaseVersion, callback); },
-    function (callback) { builder.publish(callback); },
-    function (callback) { builder.prepareNextIteration(callback); },
     function (callback) { builder.completeRelease(releaseVersion, callback); }
```

Comment:
I'm not doing `git push` and `git push --tags` as suggested by @ldez because I have multiples remotes and I want to be sure to push only on `upstream` (https://github.com/asciidoctor/asciidoctor.js).

In my case `origin` is my fork plus one remote per contributor fork.